<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pipwidget</class>
 <widget class="QWidget" name="pipwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="vbox">
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_controls" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="searchbox">
        <property name="placeholderText">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_search">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="edit-find">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_show_installed">
        <property name="text">
         <string>Show installed</string>
        </property>
        <property name="icon">
         <iconset theme="system-software-install">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_show_updates">
        <property name="text">
         <string>Show updates</string>
        </property>
        <property name="icon">
         <iconset theme="system-software-update">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_show_available">
        <property name="text">
         <string>Show available</string>
        </property>
        <property name="icon">
         <iconset theme="system-software-install"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_progress">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>dummy</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="icon">
      <iconset theme="process-stop">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
